internal/metamorph/processor_helpers.go

```diff
@@ -1,7 +1,78 @@
 package metamorph
 
-func (p *Processor) GetProcessorMapSize() int {
+import (
+	"github.com/bitcoin-sv/arc/internal/metamorph/store"
+	"github.com/libsv/go-p2p/chaincfg/chainhash"
+)
 
+func (p *Processor) GetProcessorMapSize() int {
 	return p.ProcessorResponseMap.Len()
+}
+
+func updateStatusMap(statusUpdatesMap map[chainhash.Hash]store.UpdateStatus, statusUpdate store.UpdateStatus) {
+	foundStatusUpdate, found := statusUpdatesMap[statusUpdate.Hash]
+
+	if !found || (found && shouldUpdateStatus(statusUpdate, foundStatusUpdate)) {
```

Comment:
The second `found` is not needed as it is always true
```
if !found || (shouldUpdateStatus(statusUpdate, foundStatusUpdate))
```

Reply:
You're right, fixed